### ISSUE TITLE
step

### ISSUE THREAD

**Issue by Tute05** [Tute05/Final-Project#6]
sudo: required
dist: trusty
services:
- docker

language: java
jdk: oraclejdk8

install:
  - ./gradlew clean jar

script:
  - ./gradlew test
  - echo "Testing Coming soon"

after_success:
- "./gradlew jacocoTestReport"
- find . -name jacocoTestReport.csv|xargs cat|awk -F',' '{print $3" "$4" "$5}
before_deploy:
  - ./gradlew test jacocoTestCoverageVerification
deploy:
  provider: script
  script: echo 'WIP'
  on:
    all_branches: true
**Additional context**

**Comment by Tute05** [comment]
.

**Comment by Yunir** [comment]
No, you should add here the screenshots of Cron job configuration for Travis-CI build repository.
